### ISSUE TITLE
request: rename project to `nixified-ai`

### ISSUE THREAD

**Issue by DavHau** [nixified-ai/flake#21]
The name `flake` is not optimal because:
  - after cloning the project it will end up in a directory called `flake` which might collide with other tings and also in not very expressive.
  - Also after forking the project on github it just ends up as `DavHau/flake` which has the same problems as described above.

How about just calling the project `nixified-ai/nixified-ai` ?

**Comment by MatthewCroughan** [comment]
> after cloning the project it will end up in a directory called flake

I wonder if there's a way we could fix that with Nix itself, some low level thing.

**Comment by adamcstephens** [comment]
The name of the repo is used by git when cloning. Even if I could tell Nix to rename the repo on clone, I wouldn't want to.

**Comment by MatthewCroughan** [comment]
Does anyone know how long GitHub will redirect the URL for, or whether it will at all?

**Comment by MatthewCroughan** [comment]
Right, better to do it sooner rather than later then. Can anyone think of a better name than `nixified-ai` in this case?

**Comment by DavHau** [comment]
I think the name is fine

**Comment by MatthewCroughan** [comment]
Is there any way that is known to give the Nix flake URI a default repository? So for example `nix run github:nixified-ai` would default to something we specify?

**Comment by max-privatevoid** [comment]
To that end, we could ask for a spot in the global flake registry, in which case we'd be at `nix run nixified-ai`.
